c/cmake_modules/AdbcDefines.cmake

```
@@ -95,7 +95,7 @@ if(MSVC)
   add_compile_options(/wd5027)
   add_compile_options(/wd5039)
   add_compile_options(/wd5045)
-  add_compile_options(/wd5246)
+  add_compile_options(/wd5246) 
```

Could you revert this?

```suggestion
  add_compile_options(/wd5246)
```